spec/factories/person.rb

```diff
@@ -9,6 +9,6 @@
     division2 'Command'
     title 'Captain'
     gender 'Female'
-    department 'Police'
+    association :department, factory: :department
```

Comment:
@sergeimellow : Including the association in the factory is absolutely a valid thing to do.  It just has consequences.

As @kgf alluded to, including an association in a factory makes the tests slower and more complicated under the hood.  The behavior of this association isn't critical to verify, so including it in the factory might not be worth the cost.